src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1547,6 +1547,70 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
 
 To switch to a standalone database, see <<_perf_tuning_db>>.
 
+== Nodes and Task Recovery
+
+While your tasks are running on nodes, the scheduler can experience a transient failure and come back to life without
+losing the on-going computations. This is possible thanks to the nodes and task recovery mechanism run by the scheduler.
+When the scheduler is starting, it checks the nodes and the tasks that were persisted in database to be able to recover
+on going tasks and alive nodes. For each node that is supposed to be handled by the scheduler, the recovery will consist
+into trying to contact these nodes. If a node is alive, the reference that is retrieved with the lookup of this node is
+kept by the resource manager and the internal data structures of the scheduler are updated with this node information.
+If a persisted node is not alive at recovery time, then the node will be recreated automatically.
+
+=== Scope
+
+A recovery is not a restart, and as such a clean restart will not preserve on-going tasks and running nodes. Indeed, if
+you restart the scheduler manually, it will clean and remove the nodes before exiting. Thus, there will be no node and
+no running tasks to recover when the scheduler is started again. However, in the particular case where the scheduler
+experiences a failure, no such cleaning is performed. Thus, when the scheduler starts again after a failure, it loads
+all the persisted information about the nodes state and the tasks state to be able to resume. If, for a node source, no
+node is found in database, then the node recovery will not take place for this node source. Instead, the nodes will be
+redeployed as for the first time.
+
+=== Dedicated Resource Manager property
```

Comment:
I would rather write a paragraph: Node/Task Recovery configuration

A bit less reference-book like, a bit more like "what are the important configurations necessary for Task recovery and why"